nemo/collections/nlp/models/language_modeling/megatron_bert_model.py

```diff
@@ -155,7 +155,11 @@ def training_step(self, batch, batch_idx):
             lr = self._optimizer.param_groups[0]['lr']
             self.log('lr', lr)
             self.log('global_step', self.trainer.global_step, prog_bar=True)
-            self.log('consumed_samples', self.compute_consumed_samples(self.trainer.global_step), prog_bar=True)
+            self.log(
+                'consumed_samples',
+                self.compute_consumed_samples(self.trainer.global_step - self.init_global_step),
```

Comment:
Do we want to log current run consumed samples? Perhaps global consumed samples is more meaningful?

Reply:
This is global consumed samples

Reply:
I guess a name change could make it more clear

Reply:
It is not global of the whole training, it is global for current session, right?
`self.trainer.global_step - self.init_global_step`

Reply:
It is the global of the whole training. because 
`consumed_samples = inital_loaded_consumed_samples + steps * GBS`

Reply:
Got it.